### DISCUSSION TITLE
Scrolling is too sensitive

### DISCUSSION THREAD

**Discussion by quartzbear** [sumatrapdfreader/sumatrapdf#2453] [Q&A] [answered]
Hi

When using Sumatra I read documents in "Fit Width", so I too scroll down the page too read it.
Trouble is, when I scroll the speed jumps to infinity and I end up dozens of pages ahead (or the end of the doc).
Any setting I can change to fix this?
I am using a laptop touchpad, otherwise I don't have mouse related issues with it.

Thanks

**Comment by GitHubRulesOK** [comment, marked as answer]
speed of auto-scroll is usually determined by several factors and varies for each device/user by system and hardware settings including an exponential increase decided by stroke length. 

Thus using a mouse that's usually more easy to control via middle button. So using mouse it is easy to get pixel line by line smooth slow scroll.

I don't have any auto scroll active on this new machine touchpad thus scrolling is at my request slowed by only using arrow keys and faster by using shift.

Perhaps check if there is a per app setting for your hardware?

**Reply by quartzbear** [reply]
Thanks for the advice. I'm not sure how to change the setting on the hardware for an individual program, but it's worth looking into.
Meanwhile I'll use the arrow+space bar (using the shift and arrow moves to the next page,not the next part of the current page).

I did notice something strange, when viewing in continuous mode the touchpad works well and doesn't jump to the end/beginning. It's only in the single page (width view) that I have this problem.

**Reply by GitHubRulesOK** [reply]
Ahh yes in single pages the scrollbar body usage will jump by a screens worth thus depending on  amount of page visible will be top to bottom whereas the top and bottom arrows on scrollbar move in small increments

**Reply by aleitao69** [reply]
Current Sumatra scrolling setup is working by page, thus it jumps to the next page at the end of the active page.  Is there a way to scroll by line, and in this way, I would be able to see the switching page transition?

**Reply by kjk** [reply]
There are many ways to scroll: https://www.sumatrapdfreader.org/docs/Scrolling-and-zooming

**Comment by kjk** [comment]
Compared to 3.5.2, latest [pre-release](https://www.sumatrapdfreader.org/prerelease) has improved non-continuous fit width scrolling with touchpad. It shouldn't jump between pages quickly anymore.
